### PR TITLE
Adds Pushover user attributes

### PR DESCRIPTION
**Pull request by lzkill**
The Pushover integration doesn't allow messages to be delivered per-user,
since the `NOTIFICATOR_PUSHOVER_USER` config is single for the whole instance.

### CODE REVIEW

**Comment by tananaev** [comment]
You need to fix the lint errors.

File: src/main/java/org/traccar/notificators/NotificatorPushover.java [hunk 1]
```diff
@@ -56,25 +56,24 @@ public NotificatorPushover(Config config, NotificationFormatter notificationForm
...
-        }
-
         var shortMessage = notificationFormatter.formatMessage(user, event, position, "short");
 
         Message message = new Message();
         message.token = token;
-        message.user = this.user;
-        message.device = device;
+
+        message.user = user.getString("pushoverUserKey");
```
Comment (tananaev) [inline src/main/java/org/traccar/notificators/NotificatorPushover.java:68]: I think we should reuse `NOTIFICATOR_PUSHOVER_USER` here.

Comment (lzkill) [reply src/main/java/org/traccar/notificators/NotificatorPushover.java:68]: I've used the [NotificatorTelegram.java ](https://github.com/lzkill/traccar/blob/ea68646e6572945db82bfdf8e732957492e19b03/src/main/java/org/traccar/notificators/NotificatorTelegram.java)as a guide, where the config provides a wide value that can be overwritten by an user attribute. (`NOTIFICATOR_TELEGRAM_CHAT_ID` / `telegramChatId`)

Comment (tananaev) [reply src/main/java/org/traccar/notificators/NotificatorPushover.java:68]: We can't really change what's already there, but we can enforce this for new attributes.

**Pull request merged**